pandas/core/api.py

```diff
@@ -6,6 +6,7 @@
 
 from pandas.core.algorithms import factorize, unique, value_counts
 from pandas.core.dtypes.missing import isna, isnull, notna, notnull
```

Comment:
I don't think exposing this is a good idea here. We don't directly do this with other dtypes. instead the are in ``pandas.api.types``

Reply:
Hmm, I could go either way on this. It would be the only type at the top level. On the other hand `pd.api.types.CategoricalDtype(values)` is quite long :)

I'll think about it some more, but you're probably right.